tests/utils/rpc/request.go

```diff
@@ -93,6 +94,50 @@ func Decode(body []byte, target interface{}) error {
 	return nil
 }
 
+// DecodeScale decodes []body into the target interface.
+// It assumes that the response.Result is a SCALE-encoded value.
+func DecodeScale(body []byte, target interface{}) error {
```

Comment:
nit use `any` instead of `interface{}`

```suggestion
func DecodeScale(body []byte, target any) error {
```